### DISCUSSION TITLE
Using external component libraries in TailwindCSS App

### DISCUSSION THREAD

**Discussion by c-myers1** [tailwindlabs/tailwindcss#1605] [General]
In building a TailwindCSS vuejs app, I would like to use complex widgets from other libraries that come with their own css (e.g jqwidgets or kendoui.) These typically have their own set of css to be included which I suspect would clash with tailwind’s. 

Is the best approach simply to ensure the css are scoped to those components only? Would that address issues of responsiveness of the component to tailwind’s outer boundaries per viewport etc? Is there anything else to be mindful of?

**Comment by dennzimm** [comment]
If you think there might be a collision of class names of the component library you use, you can prefix the tailwind classes

```
// tailwind.config.js
module.exports = {
  prefix: 'tw-',
}
```

https://tailwindcss.com/docs/configuration/
